packages/data-row-permissions/src/components/ServerStatus.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div
+    v-if="unauthorized">
+    <span>{{ $t('data-row-permissions-no-results-cta') }}&nbsp;<a href="/login">{{ $t('data-row-permissions-cta-login') }}</a>.</span>
```

Comment:
If the user is unauthorized, explain that this is an admin only feature. Though I think if that happens it is due to a server misconfiguration if the plugin appears in the user's menu and they can only reach it through an external link.